README.md

```diff
@@ -52,4 +51,5 @@ When running the webfrontend without Apache, you might have to manually launch w
 
 `/usr/bin/websockify -v PORT --cert=CERT --key=KEY --target-config=TOKENDIR`
 
-*NOTE: You may also need to create a `sessions/` directory*
```

Comment:
So you don't then? :confused:

Reply:
Not any more, I updated it to create it automatically if absent.

It needed to be created initially since git repos don't allow empty directories.

Reply:
Ok, could you link to the bit where that happens?

Reply:
Here you go https://github.com/stfc/cloud/pull/114/commits/eea7bc010b55713db574303a60459731de37a3fb#diff-d355af669766ee3562643e8e5034f715R21

Reply:
Thanks.